### PR TITLE
Added limitations on numeric qtypes

### PR DESCRIPTION
**Pull request by exprin**
I added the following:

- Explanation of limitations of number and decimal question types
- How to add number, range, and decimal question types in XLSForm
- a downloadable XLSForm

@joshuaberetta kindly review

### CODE REVIEW

**Review by joshuaberetta — CHANGES_REQUESTED** [review]
@exprin, is there a reason why the two files:
- source/images/pulling_data_excelquery/excel_data_source_settings.gif
- source/images/pulling_data_excelquery/excel_load_table.gif
Are included in this PR?

**Comment by exprin** [comment]
> @exprin, is there a reason why the two files:
> 
> * source/images/pulling_data_excelquery/excel_data_source_settings.gif
> * source/images/pulling_data_excelquery/excel_load_table.gif
>   Are included in this PR?

I might have accidentally added the "Pulling data into Power Query" files into master instead of creating a new branch. SO now those files are moving with me to every branch. I must have added them to the commit for this branch accidentally

File: source/number_decimal_range.md [hunk 1]
```diff
@@ -69,4 +88,71 @@ refer to the support article,
...
+You can specify advanced appearances of "Number", "Decimal" and "Range"
+questions in XLSForm under the appearances column as in the following examples:
+
+| type    | name            | label                                                    | appearance    | parameters           |
+| :------ | :-------------- | :------------------------------------------------------- | :------------ | :------------------- |
+| integer | income          | What was the total income you got in the last 12 months? | thousands-sep |                      |
+| decimal | bearing         | Capture bearing                                          | bearing       |                      |
+| range   | vertical_rating | Rate the effectiveness of the project from 1 to 5        | vertical      | start=1 end=5 step=1 |
+| range   | picker_rating   | Rate the effectiveness of the project from 1 to 5        | picker        | start=1 end=5 step=1 |
+| range   | star_rating     | Rate the effectiveness of the project from 1 to 5        | rating        | tart=1 end=5 step=1  |
```
[the remark this answers is not in the document]
Comment (exprin) [reply source/number_decimal_range.md:106]: Done

File: source/number_decimal_range.md [hunk 2]
```diff
@@ -56,6 +58,23 @@ To add a "Range" question type:
...
+
+| type    | name     | label                                             | parameters           |
+| :------ | :------- | :------------------------------------------------ | :------------------- |
+| integer | hhsize   | How many people live in this household?           |                      |
+| decimal | landsize | How big is your land? (in hectares)               |                      |
+| range   | rating   | Rate the effectiveness of the project from 1 to 5 | start=1 end=5 step=1 |
+| survey  |
+
+<p class="note">
+Note: When adding a `range` question to an xlsform, the start, end and step parameters are added in the `parameters` column.
```
[the remark this answers is not in the document]
Comment (exprin) [reply source/number_decimal_range.md:75]: Done

File: source/number_decimal_range.md [hunk 3]
```diff
@@ -69,4 +88,71 @@ refer to the support article,
...
+| range   | star_rating     | Rate the effectiveness of the project from 1 to 5        | rating        | tart=1 end=5 step=1  |
+| survey  |
+
+## Limits on “Number” and “Decimal” questions
+
+In KoboCollect:
+
+- “Number” question entries are limited to a maximum of 9 characters.
+- “Decimal” question entries are limited to a maximum of 15 characters. Note:
+  Negative signs and decimal points count towards the character limit.
```
[the remark this answers is not in the document]
Comment (exprin) [reply source/number_decimal_range.md:115]: Yes I tested this both on Enketo and Collect

**Comment by exprin** [comment]
@joshuaberetta I have sent a commit to this branch resolving the comments

**Review by joshuaberetta — APPROVED** [review]
@exprin please review the changes I made before merging.

**Pull request merged**